packages/indexer-integration-tests/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "indexer-integration-tests",
+  "version": "0.0.1",
+  "description": "Integration tests for the indexer components.",
+  "main": "index.js",
+  "repository": "https://github.com/graphprotocol/indexer",
+  "author": "Graph Protocol",
+  "scripts": {
+    "integration-test": "ts-node src/bootstrapTest.ts"
```

Comment:
I know this is basically just a placeholder for now, but I'd expect a `jest ...` command here..?

Reply:
Ah, yeah leftover of me figuring this out. Will remove.